test/unit/trim.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+var assert = require('assert');
+
+var sharp = require('../../index');
+var fixtures = require('../fixtures');
+
+describe('Trim borders', function() {
+
+  it('Threshold default', function(done) {
+    var expected = fixtures.expected('alpha-layer-1-fill-trim-resize.png');
+    sharp(fixtures.inputPngOverlayLayer1)
+      .resize(450, 322)
+      .trim()
+      .toBuffer(function(err, data, info) {
+        if (err) throw err;
+        assert.strictEqual('png', info.format);
+        assert.strictEqual(450, info.width);
+        assert.strictEqual(322, info.height);
+        fixtures.assertSimilar(expected, data, done);
+      });
+  });
+
+  it('16-bit PNG with alpha channel', function(done) {
+    sharp(fixtures.inputPngWithTransparency16bit)
+      .resize(32, 32)
+      .trim()
```

Comment:
If I change this to `trim(100)` then it will fail:
```
Uncaught Error: extract_area: parameter width not set
extract_area: parameter height not set
```

Debug:
```
(sharp:6296): GLib-GObject-WARNING **: value "-32" of type 'gint' is invalid or out of range for property 'width' of type 'gint'

(sharp:6296): GLib-GObject-WARNING **: value "-32" of type 'gint' is invalid or out of range for property 'height' of type 'gint'
[Error: extract_area: parameter width not set
extract_area: parameter height not set
]
```

It looks like only values from 1-99 are permitted.